assets/js/src/form_editor/components/header.jsx

```diff
@@ -1,13 +1,20 @@
+/* eslint-disable no-undef */
```

Comment:
Why do we need to add this? Can we get rid of it?

Reply:
It added probably editor automatically. I'll get rid.